contrib/resources/translations/README.md

```diff
@@ -0,0 +1,98 @@
+
+# General translation workflow
+
+## Preparing file with English strings
+
+First, you need an up to date file with English strings. The one present in this
+directory is most likely outdated; to generate the current one either execute:
+
+```
+dosbox -lang en -c "config -wl en.lng" -c "exit"
+```
+
+or do the following:
+
+- In the configuration file, go to the `[dosbox]` section and set`language = en`.
+- Run the most current version of DOSBox Staging.
+- Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
+  path, instead of simply `en.lng`).
```

Comment:
These steps need to be done in order, so suggest replacing bullets with a sequence.

```markdown
1. In the configuration file, go to the `[dosbox]` section and set`language = en`.
1. Run the most current version of DOSBox Staging.
1. Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
  path, instead of simply `en.lng`).
```

(the `1.` replaces the `-` and becomes a sequence at rendering time)

Reply:
OK, changed.